FirebaseSessions/Sources/SettingsCacheClient.swift

```diff
@@ -25,9 +25,9 @@ struct CacheKey: Codable {
 /// SettingsCacheClient is responsible for accessing the cache that Settings are stored in.
 protocol SettingsCacheClient {
   /// Returns in-memory cache content in O(1) time
-  var cacheContent: [String: Any]? { get }
+  var cacheContent: [String: Any]? { get set }
```

Comment:
I'm curious if we can pull this question mark off of this type by returning a default value if the `cache.dictionary` is null. It might save us some question marks in other places? Just an idea.

Reply:
Good point! Changed to return empty dictionary.